agent/command/git.go

```diff
@@ -154,20 +157,20 @@ func parseToken(token string) (string, error) {
 	return splitToken[1], nil
 }
 
-func (opts cloneOpts) getCloneCommand() ([]string, error) {
+func (opts cloneOpts) getCloneCommand(cloneParams string) ([]string, error) {
```

Comment:
Why don't we just also add clone params to opts in the opts function?

Reply:
done